tidy3d/material_library/material_library.py

```diff
@@ -80,6 +82,80 @@ def medium(self):
         return self.variants[self.default].medium
 
 
+class Variant2D(Tidy3dBaseModel):
+    """Reference, data_source, and material model for a variant of a 2D material."""
+
+    medium: Medium2D = pd.Field(
+        ...,
+        title="Material dispersion model",
+        description="A dispersive medium described by the pole-residue pair model.",
```

Comment:
I'm a little confused by this description because the type is Medium2D. Should it rather say something about the material being an anisotropic medium with pole residue models defined on the tangential dimensions of the 2D geometry? (or something like this?)